docs/LOGGING.md

```diff
@@ -78,6 +80,8 @@ The uploading is controlled on a scheduled timer. When the timer fires, the "log
 
 The "loguploader" collects stats of round-trip delay, controller CPU load percentage and log batch processing time. The current EVE implementation does not use those stats in calculating the uploading timer values.
 
+The already uploaded gzip files are put in a circular buffer in /persist/newlog/keepSentQueue directory. This directory will keep up to 1000 gzip files (maximum of 50 Mbytes of compressed log data).
```

Comment:
Why 50Mb? I thought the total size of the log directories is capped by the waittosend knob.

Doesn't the customer want a knob for the total space used and things can be in either the upload or keepsent directories?

Reply:
I have two features here:
- the circular buffer, it's the already uploaded files. will be kept up to about 1000 files, which translates to max of 50 Mbyte space, nothing to do with customer specified quota
- the user quota, is counted only for dev/app files total not exceeding that limit